### ISSUE TITLE
[GO][Classic] Chen's Empty Keg

### ISSUE THREAD

**Issue by AgeAgainst** [dalaranwow/dalaran-wow#4107]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Clicking on Chen's Empty Keg in The Barrens at 45,62 starts a cast bar that you would see when opening a container.  The container is empty.  You are unable to do this quest chain because of the empty container.

**Expected behaviour**: Tell us what should happen instead
You should get Chen's Empty keg to start the quest chain.

**Steps to reproduce the problem**:

1. Go to The Barrens at 45,62
2. Right Click on Chen's Empty keg
3. See empty loot window

**Include proofs for this behaviour**
https://s30.postimg.org/3yyb8lyf3/Chens_empty_keg.jpg
Empty container, no keg in messy bags

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.
http://www.wowhead.com/quest=819/chens-empty-keg

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

http://www.wowhead.com/quest=819/chens-empty-keg
http://wowwiki.wikia.com/wiki/Quest:Chen's_Empty_Keg
http://www.wowhead.com/object=3238/chens-empty-keg#comments (notes from GM stating quest was removed in the world Sundering before the release of cataclysm)

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
https://github.com/dalaranwow/dalaran-wow/issues/2768

**Comment by Rushor** [comment]
This works fine on PTR.

![GitHub Logo](http://i.imgur.com/X9zNmiP.jpg)
